src/pontibus/tests/utils/test_interchange_packmol.py

```diff
@@ -1708,11 +1718,69 @@ def test_solvent_nonbond_parameters(self, nonbonds, num_protein_atoms, num_water
 #    ...
 #
 #
-# def test_box_setting_dodecahedron(smc_components_benzene):
-#    ...
+@pytest.mark.parametrize("box_shape", ["dodecahedron", "cube"])
+def test_box_setting_dodecahedron(
+    box_shape,
+    smc_components_benzene_named,
+    T4_protein_component,
+    water_off,
+    protein_ff_settings,
+):
+    import openmm.unit
+
+    solvation_settings = PackmolSolvationSettings(
+        target_density=0.5 * unit.grams / unit.mL,
+        box_shape=box_shape,
+    )
+    assert solvation_settings.box_shape == box_shape
+
+    interchange, _ = interchange_packmol_creation(
+        ffsettings=protein_ff_settings,
+        solvation_settings=solvation_settings,
+        smc_components=smc_components_benzene_named,
+        protein_component=T4_protein_component,
+        solvent_component=SolventComponent(),
+        solvent_offmol=water_off,
+    )
+
+    openmm_system = interchange.to_openmm_system()
```

Comment:
Inelegant - better to test what comes through the API - but I can't figure out how best to access that and the code _should_ just do this under the hood